chainer/variable.py

```diff
@@ -867,6 +867,24 @@ def array(self, d):
             self._data[0] = d
             self._has_chainerx_array = False
 
+    def chx_view(self):
```

Comment:
How about not abbreviating chainerx -> chx and name it `chainerx_view`? This is more consistent with other APIs.

Reply:
Changed to `chainex_array` after discussion. We may consider changing `chainerx` to `chx` throughout the Chainer code afterwards (e.g. `to_chainerx` -> `to_chx`).